test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using AutoRest.TestServer.Tests.Infrastructure;
+using Azure;
+using Azure.Core;
+using dpg_customization_LowLevel;
+using dpg_customization_LowLevel.Models;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgCustomizationTest : TestServerLowLevelTestBase
+    {
+        [Test]
+        public Task GetRawModel() => Test(async (host) =>
+        {
+            var result = await new DPGClient(Key, host).GetModelAsync("raw");
+            var responseBody = JsonData.FromBytes(result.Content.ToMemory());
+            Assert.AreEqual("raw", (string)responseBody["received"]);
+        });
+
+        [Test]
+        public Task GetHandwrittenModel() => Test(async (host) =>
+        {
+            Product result = await new DPGClient(Key, host).GetModelAsync("model");
+            Assert.AreEqual("model", $"{result.Received}");
+        });
+
+        [Test]
+        public Task PostRawModel() => Test(async (host) =>
+        {
+            var value = new
+            {
+                hello = "world!"
+            };
+            var result = await new DPGClient(Key, host).PostModelAsync("raw", RequestContent.Create(value));
```

Comment:
nit: Let's use the type name instead of `var` in these tests (here and elsewhere), since the return type is important to what's happening in the test.